Library/Homebrew/env_config.rb

```diff
@@ -37,6 +37,12 @@ module EnvConfig
                      "to instead be downloaded from " \
                      "`http://localhost:8080/v2/homebrew/core/gettext/manifests/0.21`",
       },
+      HOMEBREW_API_AUTO_UPDATE_SECS:             {
+        description: "Check Homebrew's API for new formulae or cask data every " \
+                     "`HOMEBREW_API_AUTO_UPDATE_SECS` seconds. Alternatively, disable API auto-update " \
+                     "checks entirely with HOMEBREW_NO_AUTO_UPDATE.",
```

Comment:
Setting `HOMEBREW_NO_AUTO_UPDATE` doesn't actually disable API auto-updates at all, right? We'll still update it here if it's been longer than `HOMEBREW_API_AUTO_UPDATE_SECS` and the remote file has changed

Reply:
@Rylan12 I missed the `Homebrew::EnvConfig.no_auto_update?` addition, added now.

It'll now only download this file once when first needed and never update it otherwise.